connectors/pom.xml

```diff
@@ -100,6 +100,9 @@
                     <packageFileTemplate>${project.basedir}/template-connector-package.xml</packageFileTemplate>
                     <exampleSourcePropertiesFileTemplate>${project.basedir}/camel-kafka-connector-template-example-source-properties.template</exampleSourcePropertiesFileTemplate>
                     <exampleSinkPropertiesFileTemplate>${project.basedir}/camel-kafka-connector-template-example-sink-properties.template</exampleSinkPropertiesFileTemplate>
+                    <excludedComponents>
+                        <param>bonita</param>
```

Comment:
Bonita is one of the component that doesn't make sense as connector, so I choose this one to be removed.